src/mono/mono/utils/mono-threads-api.h

```diff
@@ -27,7 +27,7 @@ typedef struct _MonoStackData {
 } MonoStackData;
 
 // FIXME an ifdef to change __func__ to empty or further minimization.
-#ifndef DISABLE_LOGGING
+#if !defined(DISABLE_LOGGING) || defined(DEBUG)
```

Comment:
what if we don't disable logging in debug builds, instead? 

https://github.com/dotnet/runtime/blob/74b18d3c5e910ba2d075f075eede0543b03983fb/src/mono/mono.proj#L655

Reply:
That's probably better solution.